hack/tools/Makefile

```diff
@@ -114,7 +114,7 @@ vale: $(VALE)
 $(VALE):
 	mkdir -p $(BIN_DIR)
 	# vale uses macOS for darwin, Linux for linux and 64 for amd64, so manually map them
-	curl -sfL https://github.com/errata-ai/vale/releases/download/v$(VALE_VERSION)/vale_$(VALE_VERSION)_$(shell echo $(HOST_OS) | sed 's,darwin,macOS,g'| sed 's,linux,Linux,g')_$(shell echo $(HOST_ARCH) | sed 's,amd,,g')-bit.tar.gz | tar -xz -C /tmp/
+	curl -sfL https://github.com/errata-ai/vale/releases/download/v$(VALE_VERSION)/vale_$(VALE_VERSION)_$(shell echo $(HOST_OS) | sed 's,darwin,macOS,g'| sed 's,linux,Linux,g')_$(shell echo $(HOST_ARCH) | sed 's,amd,,g')$([ $(HOST_ARCH) = "arm64" ] && echo "" || echo "-bit").tar.gz | tar -xz -C /tmp/
```

Comment:
I believe the new logic `$([ $(HOST_ARCH) = "arm64" ] && echo "" || echo "-bit")` needs to use the `shell` command first, or else the `-bit` does not appear when I tried it for `amd64`:

`$(shell [ $(HOST_ARCH) = "arm64" ] && echo "" || echo "-bit")`

Reply:
I added the missing `shell` to the command.